tools/tests/tooltip/test.js

```diff
@@ -31,6 +31,24 @@ function removeTooltips() {
     }
 }
 
+async function checkResizeHandler(hide = false, expected = true) {
+    const sandbox = sinon.createSandbox();
+    const tooltip = document.querySelector('#smart-position');
+    sandbox.spy(tooltip, "resizeDebounced");
+
+    await tooltip.show();
+    if (hide) tooltip.hide();
+
+
+    await createAsyncSpec(() => {
+        window.dispatchEvent(new CustomEvent('resize'));
+    }, 10);
+
+    return createAsyncSpec(() => {
+        assert(tooltip.resizeDebounced.called === expected, 'The tooltip was not repositioned on window resize.')
+    }, 3);

```

Comment:
```suggestion
    });
```

They are 3 by default